packages-ts/terra-sdk/src/index.ts

```diff
@@ -8,6 +8,10 @@ export interface Round {
   epoch: number
   aggregatorRoundId: number
   observationsTS: Date
+
+  blockHeight: bigint
+  txIndex: bigint
+  txHash: string
```

Comment:
@krebernisak We could also add the timestamp, but IIUC that would require extra calls. I think I would like to keep this simple and efficient, but we could provide a helper for fetching additional info if we need to.